metrics.go

```diff
@@ -56,13 +62,17 @@ func reportFunc(fn, action string, tags ...Tags) {
 type StopTimerFunc func()
 
 func ReportFuncTiming(tags ...Tags) StopTimerFunc {
+	return reportTiming(tags...)
+}
+
+func reportTiming(tags ...Tags) StopTimerFunc {
 	clientMux.RLock()
 	defer clientMux.RUnlock()
 	if client == nil {
 		return func() {}
 	}
 	t := time.Now()
-	fn := CallerFuncName(1)
+	fn := CallerFuncName(2)
```

Comment:
Why is this change required and does it fuck up calling `ReportFuncTiming` directly?